lib/pure/uri.nim

```diff
@@ -162,21 +162,26 @@ func encodeQuery*(query: openArray[(string, string)], usePlus = true,
 
 iterator decodeQuery*(data: string): tuple[key, value: TaintedString] =
   ## Reads and decodes query string ``data`` and yields the (key, value) pairs the
-  ## data consists of.
+  ## data consists of. If compiled with -d:nimLegacyParseQueryStrict, an error
+  ## is raised when there is an unencoded ``=`` character in a decoded value,
+  ## which was the behavior in Nim < 1.6
   runnableExamples:
     import std/sugar
-    let s = collect(newSeq):
-      for k, v in decodeQuery("foo=1&bar=2"): (k, v)
-    doAssert s == @[("foo", "1"), ("bar", "2")]
+    import sequtils
```

Comment:
`import std/sequtils` works too btw, so you could've written the 1 liner `import std/[sugar,sequtils]`.

However, sugar is no longer used so please remove `import std/sugar`

Reply:
done, thanks